### ISSUE TITLE
Allow creating LetsEncrypt cert with multiple names

### ISSUE THREAD

**Issue by pswaminathan** [dnsimple/terraform-provider-dnsimple#92]
Creating a LetsEncrypt cert covering multiple names is supported by the website, and it would be nice if the terraform provider supported it as well. Currently the `name` argument only takes a single subdomain.

### Terraform Version
v1.3.7

### Affected Resource(s)
Please list the resources as a list, for example:
- dnsimple_lets_encrypt_certificate

### Terraform Configuration Files
```hcl
resource "dnsimple_lets_encrypt_certificate" "combined_tls_certificate" {
  domain_id  = var.dnsimple_domain_id
  name       = "www,someothersubdomain"
  auto_renew = true
}
```

### Expected Behavior
Single certificate covering both `www` and `someothersubdomain` is created.

### Actual Behavior
Validation error

**Comment by DXTimer** [comment]
Hi @pswaminathan thank you for taking the time to make the request, and bringing this to our attention. I am glad to let you know that we fully agree and plan to have the attribute available for configuration in the coming weeks.

**Comment by pswaminathan** [comment]
Great, thank you!

**Comment by wilson-dnsimple** [comment]
Great news @pswaminathan, we've released version 1.1.0 which has added support for alternate names on the `dnsimple_lets_encrypt_certificate` resource.
